ql/src/gen/thrift/gen-javabean/org/apache/hadoop/hive/ql/plan/api/StageType.java

```diff
@@ -30,7 +30,7 @@
   REPL_TXN(15),
   REPL_INCREMENTAL_LOAD(16),
   SCHEDULED_QUERY_MAINT(17),
-  REPL_LOAD_COMPLETE_ACK(18);
+  REPL_OPERATION_COMPLETE_ACK(18);
 
   private final int value;
 
```

Comment:
unused imports like HashMap, Util
remove the private qualifier from constructor.

Reply:
Removed.